### PR TITLE
Allow SavedModelBuilder to overwrite existing folders

### PR DESCRIPTION
**Pull request by just-a-jazz**
This adds the feature #8852 requested

### CODE REVIEW

**Comment by rohan100jain** [comment]
Thanks for doing this!

**Comment by rajhans** [comment]
thanks for adding this!

**Comment by alisidd** [comment]
No problem! :)

**Review by rohan100jain — CHANGES_REQUESTED** [review]

File: tensorflow/python/saved_model/builder_impl.py [hunk 1]
```diff
@@ -76,16 +76,25 @@ class SavedModelBuilder(object):
   ```
   """
 
-  def __init__(self, export_dir):
+  def __init__(self, export_dir, overwrite=False):
+    """Construct a SavedModelBuilder.
+
+    Args:
+      export_dir: The directory to export to
+      overwrite: If `True` overwrites if a directory already exists
```
Comment (alisidd) [inline tensorflow/python/saved_model/builder_impl.py]: Done 👍

**Review by rohan100jain — APPROVED** [review]

**Comment by rohan100jain** [comment]
@tensorflow-jenkins test this please

**Comment by martinwicke** [comment]
Sorry, we have to revert this. It is breaking some tests we require (which were sadly not run yet).

**Comment by alisidd** [comment]
@martinwicke Could you let me know what the problem was, so I could fix it?

**Comment by martinwicke** [comment]
I'm sorry the initial advice was the other way around, but we had some more discussions about this and we're very certain that we do not want to make this easy. It's a bad idea (tm), take it from people who have lost lots of work to easy overwriting. 

You can always emulate the behavior by doing the delete_recursively manually, but that I would advise against. Ideally, timestamp your directories and symlink the latest.

**Pull request merged**